openapi.json

```diff
@@ -378,6 +378,28 @@
                 "type": "integer",
                 "title": "Version",
                 "description": "The version of the prompt"
+              },
+              "model_provider": {
+                "type": "string",
+                "title": "Model Provider",
+                "description": "The model provider of the prompt",
+                "enum": ["openai", "anthropic"]
+              },
+              "model": {
+                "type": "string",
+                "title": "Model",
+                "description": "The model of the prompt"
+              },
+              "parameters": {
```

Comment:
I'm worried this won't be kept up-to-date. OpenAI changes frequently and so does Anthropic. Can we just have parameters set to any? We can link to their docs instead.

Reply:
In fact, are we type-checking this in the API? If so, we should remove it.

It's possible OpenAI has hidden kwargs that some customers know about and want to use. We should allow that.

Reply:
@jzone3 do you mean to remove the existing validations as well or just allow any additional ones?

Reply:
I think remove. The maintenance cost is too high.

Reply:
Like parameters should just be <string, any>

Reply:
@jzone3 removed validations, this is how the API looks like now
<img width="1728" alt="Screenshot 2023-09-08 at 3 44 55 PM" src="https://github.com/MagnivOrg/prompt-layer-docs/assets/20099503/f06fa429-fc04-43fc-b8d7-cce765aa4dc6">
<img width="1728" alt="Screenshot 2023-09-08 at 3 45 11 PM" src="https://github.com/MagnivOrg/prompt-layer-docs/assets/20099503/0c00598b-af43-4a40-ba23-567cde1fd4e2">